### DISCUSSION TITLE
Add Images in Tracing

### DISCUSSION THREAD

**Discussion by aiakubovich** [langfuse/langfuse#1208] [Ideas]
### Describe the feature or potential improvement

Hi. With new [GPT4 Vision](https://platform.openai.com/docs/guides/vision) model we see more and more use cases where chatbots have option to attach images to prompt. It would be super cool if langfuse would be able to track and display those images in Traces (images that attached by user or generated by LLM as ouput).

**Comment by marcklingen** [comment]
Thanks for sharing, we are hashing out how to best expand Langfuse to support multi-modal traces including audio/video or other sorts of attachments/blobs. If anyone has additional requirements/ideas on this, please share them here.

**Comment by marcklingen** [comment]
Multi-modal support was added in August: https://langfuse.com/changelog/2024-08-21-openai-multi-modal-traces

Tracking support for other modalities here (+ base64 images): https://github.com/orgs/langfuse/discussions/3004
